charts/datadog/values.yaml

```diff
@@ -75,7 +75,7 @@ datadog:
   ## Autodiscovery via Kube Service annotations is automatically enabled
   clusterChecks:
     # datadog.clusterChecks.enabled -- Enable the Cluster Checks feature on both the cluster-agents and the daemonset
-    enabled: false
```

Comment:
Should we activate CLC by default?

Reply:
no. we will activate only the cluster-agent by default